app/Console/Commands/Members/ImportMembers.php

```diff
@@ -60,6 +56,41 @@ public function handle()
         }
     }
 
+    protected function getMembers()
+    {
+        $token = 'Token '.config('vatsim-api.key');
+        $url = config('vatsim-api.base').'divisions/GBR/members';
+
+        $response = Http::withHeaders([
+            'Authorization' => $token,
+        ])->get($url)->json();
+
+        // need to iterate through pages here
+
+        $memberCollection = collect();
+
+        foreach ($response['results'] as $result) {
+            $memberCollection->push([
+                'cid' => $result['id'],
+                'rating_atc' => $result['rating'],
+                'rating_pilot' => $result['pilotrating'],
+                'name_first' => $result['name_first'],
+                'name_last' => $result['name_last'],
+                'email' => $result['email'],
+                'age_band' => $result['age'],
+                'city' => $result['countystate'],
+                'country' => $result['country'],
+                'experience' => '',
```

Comment:
Is there an argument to say these columns with empty strings can be dropped from our own DB whilst we are making these changes? 

As a separate point, do we need all of these fields?

Reply:
My aim with this PR is to match exactly what the AutoTools stuff was giving us so we aren't rewriting the entire script. 

If we get the chance to rewrite it then I totally agree we need to review things like what data we have. 

For what it's worth I don't actually think we do anything with this but I wanted to keep the structure exactly as is to be safe.